tools/prepare-source.sh

```diff
@@ -2,20 +2,20 @@
 set -eux
 
 GO_VERSION=1.21.9
+OAPI_VERSION=2.4.1
+
 GO_BINARY=$(go env GOPATH)/bin/go$GO_VERSION
-OAPI_VERSION=2.3.0
+TOOLS_PATH="$(realpath $(dirname "$0")/../bin)"
 
-# this is the official way to get a different version of golang
-# see https://go.dev/doc/manage-install
+# Install Go SDK
 go install golang.org/dl/go$GO_VERSION@latest
 $GO_BINARY download
 
 # Ensure dev tools are installed
-which goimports || $GO_BINARY install golang.org/x/tools/cmd/goimports@latest
+which goimports >/dev/null || GOBIN=$TOOLS_PATH $GO_BINARY install golang.org/x/tools/cmd/goimports@latest
```

Comment:
Will which still work here? $TOOLS_PATH wouldn't be part of the user's path.

Reply:
Oh good catch! Rebasing.